adafruit_emc2101/emc2101_lut.py

```diff
@@ -332,7 +356,7 @@ def __init__(self, i2c_bus):
 
     def initialize(self):
         """Reset the controller to an initial default configuration"""
-        self.lut_enabled = False
+        self.lut_enabled = True
```

Comment:
Is there a reason this was changed?  Does it make more sense for the LUT to be enabled by default instead of having immediate access to `manual_fan_speed`?

Reply:
I enabled it because if you're using the EMC2101_LUT class you obviously intend to use the LUT; it doesn't really make sense to then have to individually enable the LUT once you've created the instance.

If you think this a breaking change willing to revert.

Reply:
No, I agree, I think it probably makes more sense.  I'm personally fine with this change, just wanted to check about it :)